src/generate/apiCollection.ts

```diff
@@ -19,10 +19,14 @@ export class ApiCollection extends Map {
   static async init(description: {
     [key: string]: string[];
   }): Promise<ApiCollection> {
-    const apiCollection = new ApiCollection();
-    for (const group of Object.keys(description)) {
-      apiCollection.set(group, await ApiGroup.init(description[group], group));
-    }
-    return apiCollection;
+    const promises: Promise<ApiGroup>[] = Object.entries(
+      description
+    ).map(([group, paths]) => ApiGroup.init(paths, group));
+
+    return new this(...(await Promise.all(promises)));
```

Comment:
```suggestion
    return this.from(await Promise.all(promises));
```
Array constructor can have weird behavior depending on the arguments. I don't think we'd ever run into anything unexpected, but I think that using `from` is clearer.

Reply:
There are two overloads both for specifying size and doing `new ApiCollection(1)` appears to work fine in ts-node. I'm having problems with the 'from', though: `Property 'get' is missing in type 'ApiGroup[]' but required in type 'ApiCollection'.`